Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -602,7 +632,13 @@ def panorama_commit_status_command(args: dict):
 @logger
 def panorama_push_to_device_group(args: dict):
     command: str = ''
-    command += f'<device-group><entry name="{DEVICE_GROUP}"/></device-group>'
+    device_group = args.get('device-group', DEVICE_GROUP)
```

Comment:
Why this argument is needed? the device group is taken from the configuration, no? 
Also, you didn't add it in the yml file.

Reply:
It's already there, but you're right this is redundant